views/components/adaptors/index.scss

```diff
@@ -0,0 +1,4 @@
+.adaptors {
+	display: grid;
+	grid-template-columns: repeat(auto-fit, 20rem);
```

Comment:
`grid-template-columns: repeat(auto-fit, minmax(15rem, 1fr));`

On mobile this is a bit problematic, because if the viewport + padding is smaller than 20rem, the content overflows. Using this `minmax(15rem, 1fr)` means that it will shrink down to 15rem if necessary, but expand to `1fr` (a fractional unit) when there is enough space.